.github/workflows/pypi_upload.yml

```diff
@@ -26,11 +26,93 @@ jobs:
           python -m pip install --upgrade build twine
 
       - name: Build wheel and source distributions
-        run: |
-          python -m build
+        run: python -m build .
 
       - name: Upload to PyPI via Twine
         env:
           TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+        run: twine upload --verbose -u '__token__' dist/*
+
+  mypyc:
+    name: mypyc wheels (${{ matrix.name }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-20.04
+            name: linux-x86_64
+            # Black needs clang to compile successfully on Linux. -g0 disables debug symbols
+            # which cuts down the wheel size a fair bit!
+            extra_environment: "CC=clang CFLAGS='-g0 -v'"
+            extra_before_build: "&& yum install -y clang"
```

Comment:
I'm surprised this works, I don't think `yum` is an Ubuntu thing.

Reply:
Anyone serious about building wheels for Linux will use the [manylinux](https://github.com/pypa/manylinux/) docker images to help maximize compatibility. cibuildwheel uses these images which are based off Red Hat IIRC.